e2e/cypress/integration/plugins/upgrade_spec.js

```diff
@@ -42,7 +42,7 @@ describe('Plugin remains enabled when upgraded', () => {
         cy.apiRemovePluginById(pluginIdDemo);
     });
 
-    it('MM-T40 Plugin remains enabled when upgraded', () => {
+    xit('MM-T40 Plugin remains enabled when upgraded', () => {
     // * Upload Demo plugin from the browser
         const fileName1 = 'com.mattermost.demo-plugin-0.1.0.tar.gz';
         const fileName2 = 'com.mattermost.demo-plugin-0.2.0.tar.gz';
```

Comment:
Seems like here we are not using `cy.apiInstallPluginFromUrl` because the point is deliberately to test the UI. 
This means the test will never be able to run green without human intervention. 

@saturninoabril not sure how to solve this problem. Can we commit the tar.gz files containing the binaries directly to the repo so they already exist in `fixtures`? This would work but seems not ideal.

Reply:
My main motivation behing using  `cy.apiInstallPluginFromUrl` is that we don't have to commit the binaries then.

Reply:
We're not actually including plugin binaries to the repo. On top of this file, there's an instruction on how to download and save the binaries for local testing/dev. In CI, there's a separate step/job that handles automatic downloads of required binaries.